qi-lib/flow/extended/expander.rkt

```diff
@@ -182,6 +182,10 @@ core language's use of #%app, etc.).
     clos
     (clos onex:closed-floe)
     (esc ex:racket-expr)
+    (~> ((~literal lambda) e ...)
+        #'(esc (lambda e ...)))
+    (~> ((~literal λ) e ...)
+        #'(lambda e ...))
```

Comment:
It's not clear to me what `e` matches here: flow syntax, racket syntax,
anything? (I think the PR description mentions some confusion about which syntax
is being written, but I don't remember.) Perhaps `e` should be annotated?

Are there accompanying docs?

Reply:
I don't think we want an annotation here as we're saying "lambda <anything>" should be delegated to the host language expander. If we did any validation here, I believe we would be doing the Racket expander's job. E.g. if the user happened to provide something other than an `expr` where an `expr` is expected in Racket's lambda, we'd like the error to come from Racket rather than from Qi.

Reply:
I've added docs!